website/tools/minimize-language.js

```diff
@@ -5,9 +5,39 @@ import {
   existsSync,
 } from 'fs';
 
-for (const folder of readdirSync('./dist/locales', 'utf8')) {
-    if (existsSync('./dist/locales/' + folder + '/translation.json')) {
-      const content = readFileSync('./dist/locales/' + folder + '/translation.json', 'utf8',);
-      writeFileSync('./dist/locales/' + folder + '/translation.json', JSON.stringify(JSON.parse(content,),),);
+import yaml from 'yaml';
+import { dirname } from 'path';
+
+const generateJSONFromYAML = (yamlPath, outputPath) => {
+  if (existsSync(yamlPath)) {
```

Comment:
I don't think it's needed, the only time this will not be true, is if someone manages to delete the lang file in the ns between reading the dir and this executing